spring-kafka-docs/src/main/kotlin/org/springframework/kafka/kdocs/started/consumer/Application.kt

```diff
@@ -44,7 +47,10 @@ class Application {
         println(`in`)
     }
 
-    fun main(args: Array<String>) = runApplication<Application>(*args)
+    companion object {
+        @JvmStatic
```

Comment:
Why do we need this?
What happened with the previous version?

Reply:
OK. I found the problem. The `main()` method must be declared outside of the class. It has to be a top level function. Plus like this:
```
fun main(args: Array<String>) {
    runApplication<Application>(*args)
}
```

returning `Unit`.

So, or this, or let's leave like yours! 😄 

Sorry for errors: I should double checked before committing such a polishing.